### ISSUE TITLE
JAVA_HOME always set to JRE when not explicitly set

### ISSUE THREAD

**Issue by drewr** [elastic/runbld#18]
We don't need https://github.com/elastic/runbld/blob/master/src/clj/runbld/java.clj#L33.

**Comment by drewr** [comment]
Nope, this is actually #19.

**Comment by drewr** [comment]
OK, I'm not sure what's going on. The `java.home` system property seems to always be `.../jre` regardless of the classpath. Tracking here.

**Comment by drewr** [comment]
For documentation purposes, `/etc/alternatives/java` was pointing to the JRE:
```
% ls -l /etc/alternatives/java
lrwxrwxrwx 1 root root 46 Sep  3 14:30 /etc/alternatives/java -> /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
```

But I've since `install`ed the JDK path as a link:

```
% sudo update-alternatives --install /usr/bin/java java /usr/lib/jvm/java-8-openjdk-amd64/bin/java 1000
% sudo update-alternatives --config java
% ls -l /etc/alternatives/java
lrwxrwxrwx 1 root root 42 Jan  5 10:49 /etc/alternatives/java -> /usr/lib/jvm/java-8-openjdk-amd64/bin/java
```

And this problem still happens. What we do here is run a separate JVM to inject some code and grab the properties we care about (we don't want it from runbld in case they're different).

**Comment by drewr** [comment]
Closed in 6faf772982e090414e0a8caa664fcc1ce45fb6ca, which erroneously mentions #17 :-1:
